index.html

```diff
@@ -0,0 +1,201 @@
+                        <!-- <i class="fa-light fa-heart fa-4x"></i> -->
+                        <!-- <span id="myButton"><i class="fa-regular fa-heart"></i></span> -->
+                        <a href="https://kit.fontawesome.com/a462069d14.js" crossorigin="anonymous"><i class="fa-regular fa-heart"></i></a>
+                    </div>
+            </div>
+            <div>
+                <img src="pexels-valeria-boltneva-1639562.jpg" alt="tantalizing burger">
+                <h2>Burger King</h2>
+                <h2>KSH 400.00</h2>
+                <p>Grilled burger with tripple cheese and lettuce</p>
+                <hr>
+                    <p>
+                        Ingredients and Instructions<span id="dots">...</span>
+                        <span id="more">
+                        <h4>Ingredients</h4>
+                            <ul>
+                                <li>1/2 teaspoon olive oil</li>
+                                <li>1 onion, peeled and fineely chopped</li>
+                                <li>1 500gm pack British beef steak mince </li>
+                                <li>1 teaspoon mixed dried herbs</li>
+                                <li>1 egg beaten</li>
+                                <li>4 slices mature Cheddar</li>
+                                <li>4 white rolls</li>
+                                <li>few round lettuce leaves, torn</li>
+                                <li>1 beef tomato, sliced</li>
+                                <li>Ketchup, to serve(optional)</li>
+                            </ul>
+                            <h4>Instructions/Method</h4>
+                            <p>
+                                Heat the olive oil in a frying pan, add the onion and cook for 5 minutes until softened and starting to turn golden. Set aside.
+                                In a bowl, combine the beef mince with the herbs and the egg. Season, add the onions and mix well. Using your hands, shape into 4 patties.
+                                Cook the burgers on a preheated barbecue or griddle for 5-6 minutes on each side. While the second side is cooking, lay a slice of cheese on top to melt slightly (if using).
+                                Meanwhile, lightly toast the cut-sides of the buns on the barbecue. Fill with the lettuce, burgers and tomato slices. Serve with ketchup, if you like.
+                            </p>
+                        </span>
+                    </p>
+                    <div class="buttons">
+                        <button onclick="myFunction()" id="myButton">View More</button>
+                        <!-- <i class="fa-regular fa-heart"></i> -->
+                        <a href="https://kit.fontawesome.com/a462069d14.js" crossorigin="anonymous"><i class="fa-regular fa-heart"></i></a>
+                    </div>
+            </div>
+            <div>
+                <img src="pexels-dzenina-lukac-1583884.jpg" alt="crispy chips">
```

Comment:
its good practice to have images in its own folder because assumming your project will have thousands of images zikikaa njee zitakua mob sana na ngumu kumanage

create a new folder assets then ndani folder ingine images

add all your images apo kwa hiyo folder and link them penye unazitumia as src="./assets/images/pexels-valeria-boltneva-1639562.jpg"